test/include.am

```diff
@@ -58,25 +58,25 @@ LOG_COMPILER = true
 TESTS_COMPILER = true
 
 test_runner_epoll: test/test.sh
-	test/test.sh -b EPOLL
+	$(srcdir)/test/test.sh -b EPOLL
```

Comment:
It will `chdir` into `/path/to/libevent` anyway, no?
And I just checked it, and it works `make check` even without `$(srcdir)`, am I missing something?